test/blackbox-tests/test-cases/tests-stanza/dune

```diff
@@ -0,0 +1,2 @@
+(tests
+ (names bar foo))
```

Comment:
what do you think about using the "expect" and "regular" names for these two tests?

Reply:
So have 2 fields? E.g.

```
(tests
 (regular foo bar)
 (expect baz blue))
```

Yeah, I definitely like this proposal. Though I would switch regular to something like `run`. I think @diml has a preference for auto discovery here.

Reply:
Oops. I renamed all the tests. Okay, auto discovery it is.